device/dell/x86_64-dell_s6100_c2538-r0/plugins/psuutil.py

```diff
@@ -8,7 +8,7 @@
 
 try:
     from sonic_psu.psu_base import PsuBase
-except ImportError as e:
+except ImportError, e:
```

Comment:
Why change from `as` to comma? The "comma variable" is no longer supported by Python 3.x, and this plugin is imported by both Python 2.7 and 3.6 scripts. Please change back to `as`.